contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -88,50 +98,48 @@ contract OETHVaultCore is VaultCore {
     }
 
     // @inheritdoc VaultCore
-    function _redeem(uint256 _amount, uint256) internal virtual override {
+    function _redeem(uint256 _amount, uint256 _minimumUnitAmount)
+        internal
+        virtual
+        override
+    {
         // Override `VaultCore._redeem` to simplify it. Gets rid of oracle
         // usage and looping through all assets for LST-mix redeem. Instead
-        // does a simple WETH-only redeem with zero fee.
+        // does a simple WETH-only redeem.
         emit Redeem(msg.sender, _amount);
 
-        if (IERC20(weth).balanceOf(address(this)) >= _amount) {
+        require(_amount > 0, "Amount must be greater than 0");
```

Comment:
We don't want to change behavior here from the past code and revert where we didn't revert before - can you replace this require with an `if(_amount==0){ return; }` ?